docs/podman-create.1.md

```diff
@@ -804,13 +804,28 @@ KillMode=process
 WantedBy=multi-user.target
 ```
 
+### Rootless Containers
+
+Podman runs as a non root user on most system. This feature requires that a new enough version of shadow-utils
+be installed.  The shadow-utils package must include the newuidmap and newgidmap executables.
```

Comment:
Can (should) we include the minimum version of shadow-utils somewhere in here?

Reply:
I did not want to, since this varies by packaging.  Debian version versus rpm...

Reply:
Just saw this flying in via email: s/most system/most systems/, s/be installed/is installed/

Reply:
Fixed